src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -509,6 +509,16 @@ private function addSessionSection(ArrayNodeDefinition $rootNode)
                             ->defaultValue(0)
                             ->info('seconds to wait between 2 session metadata updates')
                         ->end()
+                        ->integerNode('sid_length')
+                            ->defaultValue(32)
```

Comment:
I don't think that we should define a default here. If no value is provided, the fallback should be whatever has been set via php.ini.

Reply:
Indeed, I've removed it but not push the correction right now